hypervisor/src/mshv/mod.rs

```diff
@@ -807,6 +814,17 @@ impl vm::Vm for MshvVm {
 
     /// Creates a guest physical memory region.
     fn create_user_memory_region(&self, user_memory_region: MemoryRegion) -> vm::Result<()> {
+        // No matter read only or not we keep track the slots.
+        // For readonly hypervisor can enable the dirty bits,
+        // but a VM exit happens before setting the dirty bits
+        self.dirty_log_slots.write().unwrap().insert(
```

Comment:
Please remove this and the other `unwrap` in `remove_user_memory_region` in a follow-up PR.

Reply:
Throughout the code we consider it acceptable to `.unwrap()` on a `MutexGuard` or `RwLockGuard`. panic()ing on there is the best cause of action since a user won't be able to do anything useful with an error from that.

Reply:
Fair enough. In that case, there is no need to change this.